docker/.env.backend.example

```
@@ -0,0 +1,24 @@
+LANGUAGE_TOOL_API_URL=http://languagetool:8010/v2
+LANGUAGE_TOOL_CACHE_DIR=~/.cache/languagetool
+PORT=8000
+
+LOG_LEVEL=debug
+
+AZURE_CLIENT_ID=YOUR_TOKEN_HERE
+AZURE_TENANT_ID=YOUR_TOKEN_HERE
+AZURE_FRONTEND_CLIENT_ID=YOUR_TOKEN_HERE
+SCOPE_DESCRIPTION="user_impersonation" # The scope name of the Azure Entra Backend API App
+
+HMAC_SECRET=YOUR_GENERATED_SECRET # Used to pseudonymize user id. Create with openssl rand 32 | base64
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)